opentracing-spring-web-autoconfigure/src/main/java/io/opentracing/contrib/spring/web/autoconfig/WebTracingConfiguration.java

```diff
@@ -0,0 +1,33 @@
+package io.opentracing.contrib.spring.web.autoconfig;
+
+import java.util.regex.Pattern;
+
+/**
+ * @author Pavol Loffay
+ */
+public class WebTracingConfiguration {
+    public static final Pattern DEFAULT_SKIP_PATTERN = Pattern.compile("/health|/favicon\\.ico");
+
+    public final Pattern skipPattern;
```

Comment:
I have changed it to getter. The reason for this is that java is too verbose for this simple props.